atest/robot/standard_libraries/builtin/set_test_message_and_test_message_variable.robot

```diff
@@ -0,0 +1,17 @@
+*** Settings ***
+Documentation     Unit test suite to Utils.Report_Failure_Due_To_Bug keyword.
+
+*** Test cases ***
+Message_Contains_Reported_Bug
+    Pass_Execution    Initial Test Message
+    [Teardown]    Check_Test_Message_Behavior
+
+*** Keywords ***
+Check_Test_Message_Behavior
+    Should_Be_Equal    ${TEST_MESSAGE}    Initial Test Message
+    BuiltIn.Set_Test_Message    First    True
```

Comment:
No need to use `_` in test and keyword names nor `BuiltIn.` prefix when calling `Set Test Message`.